src/slowmokit/methods/metrics/silhouette_score.hpp

```diff
@@ -0,0 +1,15 @@
+/**
+ * @file methods/metrics/silhoutte_score.hpp
+ *
+ * Easy include to find Silhoutte score
+ */
+
+#ifndef SLOWMOKIT_SILHOUETTE_SCORE_HPP
+#define SLOWMOKIT_SILHOUETTE_SCORE_HPP
+#include "../../core.hpp"
+
+template<class T>
+double SilhouetteScore(std::vector<std::vector<T>> x, std::vector<int> y,
+                       int numClusters, std::string typeDist);
```

Comment:
- Add `doc comments` explaining each parameter and what function does.
- Also no need to add variable names in signature, just the type is enough.
   - `double SilhouetteScore(std::vector<std::vector<T>>, std::vector<int>,
                       int, std::string);` 
- Pass the parameter as `const` if you are not going to change them so compiler optimizes the code.